src/main.c

```diff
@@ -49,6 +49,7 @@ main (int argc, char *argv[])
   };
 
   short int next_option = 0;
+  int rmwed_files = 0;
```

Comment:
Here's a tip they don't mention the C textbooks: declare variables near where they are first  used.

And this case, that can become an "[automatic variable](https://en.wikipedia.org/wiki/Automatic_variable)" by declaring it around L189, between the if and the for.